integration_test/scenarios/search_suggestion_highlights_scenario.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:core/core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../base/base_scenario.dart';
+import '../models/provisioning_email.dart';
+import '../robots/search_robot.dart';
+import '../robots/thread_robot.dart';
+import '../utils/scenario_utils_mixin.dart';
+import 'login_with_basic_auth_scenario.dart';
+
+class SearchSuggestionHighlightsScenario extends BaseScenario with ScenarioUtilsMixin {
+  SearchSuggestionHighlightsScenario(
+    super.$, {
+    required this.loginWithBasicAuthScenario,
+    required this.keyword,
+    required this.longContentWithSearchKeywordAtTheStart,
+    required this.longContentWithSearchKeywordAtTheMiddle,
+    required this.longContentWithSearchKeywordAtTheEnd,
+  });
+
+  final LoginWithBasicAuthScenario loginWithBasicAuthScenario;
+  final String keyword;
+  final String longContentWithSearchKeywordAtTheStart;
+  final String longContentWithSearchKeywordAtTheMiddle;
+  final String longContentWithSearchKeywordAtTheEnd;
+
+  @override
+  Future<void> execute() async {
+    final longEmailContents = [
+      longContentWithSearchKeywordAtTheStart,
+      longContentWithSearchKeywordAtTheMiddle,
+      longContentWithSearchKeywordAtTheEnd
+    ];
+
+    // Robots
+    final threadRobot = ThreadRobot($);
+    final searchRobot = SearchRobot($);
+
+    // Login
+    await loginWithBasicAuthScenario.execute();
+
+    // Prepare attachment file
+    final file = await preparingTxtFile(keyword);
+
+    // Provisioning some emails
+    await provisionEmail(longEmailContents
+      .map(
+        (emailContent) => ProvisioningEmail(
+          toEmail: loginWithBasicAuthScenario.email,
+          subject: keyword,
+          content: emailContent,
+          attachmentPaths: emailContent == longContentWithSearchKeywordAtTheMiddle
+            ? [file.path]
+            : [],
+        ),
+      )
+      .toList());
+    await $.pumpAndTrySettle(duration: const Duration(seconds: 5));
```

Comment:
` Why do we use`pumpAndTrySettle` and not `pump ` or `pumpAndSettle`?

Reply:
Replaced with `pumpAndSettle`